File-Formats/PDF/Converting-HTML-to-PDF.md

```diff
@@ -183,12 +183,12 @@ Integrating HTML to PDF converter library in any .NET application is simple. Ple
 <b>Steps to convert HTML to PDF in .NET application</b>
 
 Step 1: Create a new .NET console application.
-<img src="Convert-HTML-To-PDF/htmlconversion_images/createconsolesample1.png" alt="Create .net core console sample" width="100%" Height="Auto"/>
+<img src="convert-html-to-pdf/htmlconversion_images/create-net-core-console-sample.png" alt="Create .NET Core core console sample." width="100%" Height="Auto"/>
```

Comment:
Use this - Create .NET Core console sample.

Reply:
updated